db/migrate/20170414233058_create_poems.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePoems < ActiveRecord::Migration[5.0]
+  def up
+    create_table :poems do |t|
+      t.string :title, null: false
+      t.string :status, default: "Pending", null: false
+      t.references :user, index: true, foreign_key: true, null: false
+    end
+  end
+
+  def down
+    drop_table :poems
+  end
```

Comment:
sounds good. I wasn't sure if we just steered clear of `change` in favor for the added control.